src/cic/model.clj

```diff
@@ -51,7 +52,10 @@
                               a (get model-coefs (str "admission_age" age) 0.0)
                               b (get model-coefs "quarter")
                               c (get model-coefs (str "quarter:admission_age" age) 0.0)
-                              y (m/exp (+ intercept a (* b day) (* c day)))
+                              y (try (m/exp (+ intercept a (* b day) (* c day)))
+                                     (catch Exception e
+                                       (print intercept a age model-coefs)
+                                       (throw e)))
```

Comment:
throw this as an ex-info with the msg from the print and the causing exception

Reply:
`(try (m/exp (+ intercept a (* b day) (* c day)))
                                     (let [message (print intercept a age model-coefs)]
                                       (catch Exception e message)
                                         (throw (ex-info message {:err e})))`

Reply:
What you had in mind?

Reply:
yes, but print will return nil rather than the string, so I'd perhaps use `format`